bundles/org.openhab.binding.mielecloud/src/main/resources/OH-INF/thing/dishwasherDevice.xml

```diff
@@ -34,6 +34,8 @@
 			<channel id="error_state" typeId="error_state"/>
 			<channel id="info_state" typeId="info_state"/>
 			<channel id="door_state" typeId="door_state"/>
+			<channel id="water_consumption_current" typeId="water_consumption_current"/>
```

Comment:
Please make sure to add proper update instructions to allow updating managed things. See https://github.com/openhab/openhab-core/pull/3330

Reply:
Is there any developer documentation on how to do this apart from the examples from the PR?

Reply:
Not yet. But you also could have a look at the PR I just submitted for some channels I added to the sleepiq binding.

https://github.com/openhab/openhab-addons/pull/14460

Reply:
Is it required to add the `thingTypeVersion` property for each thing type with update instructions?

```
		<properties>
			<property name="thingTypeVersion">1</property>
		</properties>
```

Does this serve the purpose of preventing updates on newly created things?